dockerfile/dockerfile

```diff
@@ -24,16 +24,16 @@ ARG KUBE_VERSION
 ARG HELM_VERSION
 
 ENV KUBE_VERSION ${KUBE_VERSION:-v1.14.8}
-ENV HELM_VERSION ${HELM_VERSION:-v2.15.0}
-ENV HELM_DIFF_VERSION ${HELM_DIFF_VERSION:-v2.11.0+5}
+ENV HELM_VERSION ${HELM_VERSION:-v3.0.0-rc.1}
+ENV HELM_DIFF_VERSION ${HELM_DIFF_VERSION:-v3.0.0-rc.7}
 
 RUN apk --no-cache update \
     && apk add --update --no-cache ca-certificates git openssh ruby \
     && apk add --update -t deps curl tar gzip make bash \
     && rm -rf /var/cache/apk/* \
     && curl -L https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
     && chmod +x /usr/local/bin/kubectl \
-    && curl -L http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar zxv -C /tmp \
+    && curl -L https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar zxv -C /tmp \
```

Comment:
Just by tweaking a little bit the `curl` command maybe you can remove lines 37, 38 and 39
```
curl -L https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz \
  | tar xzf - --strip-components=1 -C /usr/local/bin/ linux-amd64/helm
```
That will extract only the Helm binary into `/usr/local/bin/`

Actually the way it is now is more explicit than my command

Reply:
I'll let myself leave it as it is, don't want to spend time on sugar at this time, I'd rather like to reach RC state asap :-)